packages/web/pages/apps/index.tsx

```diff
@@ -0,0 +1,168 @@
+import Fuse from "fuse.js";
```

Comment:
Instead of nesting the `index` route, we could flatten it by creating `apps.tsx` within `packages/web/pages/apps`. This makes it more discoverable by VS Code file search

Reply:
Yes, Next will work with both structures. This is just a suggestion to improve VSCode search results. If you search for pools, the editor has trouble finding the `pages` index file that is nested within `pools`